packages/css/src/plugins/normalize-css.tsx

```diff
@@ -0,0 +1,72 @@
+import cssnano from 'cssnano-preset-default';
+import { Plugin } from 'postcss';
+import { normalizeCurrentColor } from './normalize-current-color';
+
+/**
+ * These plugins are ran on production builds to ensure the minimal amount of CSS is generated.
+ */
+const PROD_PLUGINS = [
+  // Order CSS declaration values so any combination results in the same atomic rule.
+  'postcss-ordered-values',
+
+  // Replace values to equivalent smaller values.
+  'postcss-reduce-initial',
+  'postcss-convert-values',
+
+  // Normalize color values
+  'postcss-colormin',
+
+  // Normalize specific values so they are all equivalent.
+  'postcss-normalize-url',
+  'postcss-normalize-unicode',
+  'postcss-normalize-string',
+  'postcss-normalize-positions',
+  'postcss-normalize-timing-functions',
+
+  // Make gradients as small as possible.
+  'postcss-minify-gradients',
+
+  // Throw away comments
+  'postcss-discard-comments',
+
+  // Reduce CSS calc expressions if possible.
+  'postcss-calc',
+];
+
+/**
+ * These plugins are always ran on any run.
+ */
+const BASE_PLUGINS: string[] = [
+  // This ensures all selectors are the same even with different white space
+  'postcss-minify-selectors',
+
+  // This ensures all at rules are the same even with different white space
+  'postcss-minify-params',
+];
+
+/**
+ * This plugin runs a subset of the cssnao plugins to normalize CSS during build.
+ * During a production build it will run more plugins.
+ */
+export const normalizeCSS = (): Plugin<never>[] => {
+  const preset = cssnano();
+  // We exclude async because we need this to run synchronously as ts transformers aren't async!
+  const extraPlugins = process.env.NODE_ENV === 'production' ? PROD_PLUGINS : [];
```

Comment:
current logic: we don't want people to "opt in" to this behaviour. thats why we force it during production builds.